### DISCUSSION TITLE
AEM Modernize Tool - Bucket List-Status - Not Found

### DISCUSSION THREAD

**Discussion by nagendra-pothula** [adobe/aem-modernize-tools#212] [Q&A]
I am trying to convert geometrixx site using Modernize tool. I added basic rules for title and text components conversion and updated Modernize configurations with correct path. 

When I trigger job for Component Conversion, I am getting Status as "Not found". There are no errors in Modernize logs to debug issue.
Even though rules are working fine, I am able to see count as well before triggering job but still no luck

<img width="1786" alt="image" src="https://github.com/adobe/aem-modernize-tools/assets/65727190/ed8a2d05-8e2c-4f9f-9920-3c46d77d84c4">

<img width="1558" alt="image" src="https://github.com/adobe/aem-modernize-tools/assets/65727190/ceaf5d86-0030-4ae7-bf73-78421866f076">

Please help me here as I am stuck with no where to find the issue. Its greatly appreciated your help

**Comment by bstopp** [comment]
Can you post your transformation rules and OSGI configurations here?
